test/config/utility.cc

```diff
@@ -369,7 +373,113 @@ envoy::config::cluster::v3::Cluster ConfigHelper::buildCluster(const std::string
       http2_protocol_options: {{}}
     )EOF",
                                                                                  name, name,
-                                                                                 ip_version, port));
+                                                                                 address, port));
+}
+
+envoy::config::cluster::v3::Cluster ConfigHelper::buildCluster(const std::string& name) {
+  return TestUtility::parseYaml<envoy::config::cluster::v3::Cluster>(fmt::format(R"EOF(
+      name: {}
+      connect_timeout: 5s
+      type: EDS
+      eds_cluster_config: {{ eds_config: {{ ads: {{}} }} }}
+      lb_policy: ROUND_ROBIN
+      http2_protocol_options: {{}}
+    )EOF",
+                                                                                 name));
+}
+
+envoy::config::cluster::v3::Cluster ConfigHelper::buildRedisCluster(const std::string& name) {
```

Comment:
suggest combining the two buildCluster/buildRedistCluster and using lb_policy default ROUND_ROBIN. 

The other difference is http2_protocol_options, which are the defaults in both.